ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/CaseEvent.java

```diff
@@ -5,7 +5,7 @@
     SUBMIT_CLAIM("submitClaimEvent"),
     DEFAULT_CCJ_REQUESTED("DefaultCCJRequested"),
     DEFENCE_SUBMITTED("DefenceSubmitted"),
-    MORE_TIME_REQUESTED("MoreTimeRequested"),
+    MORE_TIME_REQUESTED_ONLINE("MoreTimeRequestedOnline"),
```

Comment:
Don't a paper event as well?

Reply:
not really currently, as we aren't sending it or processing these.
i.e. ccd calls us with it

Reply:
I've made the callback controller more generic so it is needed now